docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -32,7 +32,6 @@ refs:
 # Configure recording rules
 
 {{< admonition type="note" >}}
-In Grafana Cloud, you can only create data source-managed recording rules.
 
 In Grafana OSS and Enterprise, you can create both Grafana-managed and data source-managed recording rules if you enable the `grafanaManagedRecordingRules` feature flag.
```

Comment:
Should it be explicit that GMA RR are available in Cloud too? I mean "In Grafana OSS, Enterprise, and Cloud, you can..."

Reply:
💯, and the feature toggle is only for Grafana-managed. I'm pushing a commit with a change for it.